keyboards/toril940/mariviper_v23/info.json

```diff
@@ -0,0 +1,997 @@
+{
+    "manufacturer": "Maria",
+    "keyboard_name": "MariViper v2/v3",
+    "maintainer": "toril940",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight" : true,
+        "led_matrix": true
+    },
+    "indicators": {
+        "caps_lock": "B11",
+        "on_state": 0
+    },
+    "rgblight": {
+        "led_count": 25,
+        "animations": {
+            "static_light": true,
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "snake": true,
+            "christmas": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "driver": "ws2812",
+        "sleep": true
+    },
+    "ws2812": {
+        "driver": "pwm",
+        "pin": "B9"
+    },
+    "led_matrix": {
+        "driver": "is31fl3731",
+        "layout": [
+            {"x": 0, "y": -1, "flags": 8},
+            {"x": 1, "y": -1, "flags": 8},
+            {"x": 2, "y": -1, "flags": 8},
+            {"x": 3, "y": -1, "flags": 8},
+            {"x": 4, "y": -1, "flags": 8},
+            {"x": 5, "y": -1, "flags": 8},
+            {"x": 6, "y": -1, "flags": 8},
+            {"x": 7, "y": -1, "flags": 8}
```

Comment:
Negative Y positions are invalid.

Reply:
Fixed to 0! Not really sure what to do when they're actually physically above the keys, but 0 should work right?